### PR TITLE
Steps quiz - fix content and accessibility issue

### PR DESCRIPTION
**Pull request by Ricokola**
Description:
- Fix minor content issue on language toggle.
- Further to an accessibility assessment, in order to meet WCAG criterias 1.3.1, 3.3.2 and 4.1.2, the form element `<progess>` must be accompanied or wrapped by a `<label>` element : [https://developer.mozilla.org/en-US/docs/Web/HTML/Element/progress#accessibility](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/progress#accessibility).

### CODE REVIEW

**Comment by Garneauma** [comment]
Pre-approve upon addition of accessibility report and méli-mélo checklist review.

**Comment by Ricokola** [comment]
> Pre-approve upon addition of accessibility report and méli-mélo checklist review.

@Garneauma changes and additions have been made and pushed to PR. Please review and let me know if it should require any additional changes.

**Comment by Garneauma** [comment]
@Ricokola Could you please translate the working example page so we can test the French version?

File: méli-mélo/2024-04-stepsquiz/reports/a11y-1-en.html [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+			</ul>
+		</dd>
+		<dt>Toward a conformity at:</dt>
+		<dd>WCAG 2.1 (Option: Level AA)</dd>
+		<dt>Summary:</dt>
+		<dd>Partial evaluation focused only on the component not its container. Evaluated all Success Criteria at Level AA of WCAG 2.2</dd>
+	</dl>
+
+	<h3 id="acr">Assessment details</h3>
+	<table class="table table-hover table-bordered" cellspacing="0" rules="all" border="1" id="MainContent_FormView_GridViewPoints" style="border-collapse:collapse;">
```
Comment (duboisp) [inline méli-mélo/2024-04-stepsquiz/reports/a11y-1-en.html:48]:
```suggestion
	<table class="table table-hover table-bordered" id="MainContent_FormView_GridViewPoints">
```

File: méli-mélo/2024-04-stepsquiz/reports/a11y-1-en.html [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+		<tbody
+			<tr>
+				<th scope="col">Standard Name</th><th scope="col">Point Name</th>
+			</tr>
```
Comment (duboisp) [inline méli-mélo/2024-04-stepsquiz/reports/a11y-1-en.html:52]:
```suggestion
		<thead>
			<tr>
				<th>Standard Name</th><th>Point Name</th>
			</tr>
		</thead>
		<tbody>
```

File: méli-mélo/2024-04-stepsquiz/reports/a11y-1-en.html [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+			<tr>
+				<td>Web Content Accessibility Guidelines</td><td>3.3.2 Labels or Instructions (Level A)</td>
+			</tr>
+			<tr>
+				<td>Web Content Accessibility Guidelines</td><td>4.1.2 Name, Role, Value (Level A)</td>
+			</tr>
+		</tbody>
+	</table>
+	<dl class="dl-horizontal">
```
Comment (duboisp) [inline méli-mélo/2024-04-stepsquiz/reports/a11y-1-en.html:64]: Let add a sub heading for readability
```suggestion
	<h4>Error 1 - Form elements must have labels</h4>
	<dl class="dl-horizontal">
```

**Review by duboisp — DISMISSED** [review]

File: méli-mélo/2024-04-stepsquiz/reports/a11y-1-en.html [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+<nav>
+	<h3>On this page</h3>
+	<ul>
+		<li><a href="#summary">Summary</a></li>
+		<li><a href="#acr">Assessement details</a></li>
+	</ul>
+</nav>
+
+<section>
+	<h3 id="summary">Summary</h2>
```
[the remark this answers is not in the document]
Comment (Ricokola) [reply méli-mélo/2024-04-stepsquiz/reports/a11y-1-en.html:22]: @duboisp sub-headings under "On this page" should be H3 correct?

**Comment by Ricokola** [comment]
@Garneauma Changes identified by @duboisp has been made. Note that I have added a `<span class="wb-inv">Questionnaire progress:</span>` for the label text so it's now showing on the page.

**Review by Garneauma — APPROVED** [review]

**Pull request merged**